coderd/apikey.go

```diff
@@ -306,16 +306,24 @@ func (api *API) tokens(rw http.ResponseWriter, r *http.Request) {
 // @Router /users/{user}/keys/{keyid} [delete]
 func (api *API) deleteAPIKey(rw http.ResponseWriter, r *http.Request) {
 	var (
-		ctx   = r.Context()
-		keyID = chi.URLParam(r, "keyid")
+		ctx               = r.Context()
+		keyID             = chi.URLParam(r, "keyid")
+		auditor           = api.Auditor.Load()
+		aReq, commitAudit = audit.InitRequest[database.APIKey](rw, &audit.RequestParams{
```

Comment:
👀

Reply:
Huh this came from the merge. Idk why it is a diff :thinking:

Reply:
This is in main :thinking: 

https://github.com/coder/coder/blob/f7842ee1a292e2db767be7f08f145e0b210e99e4/coderd/apikey.go#L308-L319